toronto/src/components/molecules/Upload/index.js

```diff
@@ -0,0 +1,117 @@
+import styled from 'styled-components';
+import { useRef, useState } from 'react';
+import PropTypes from 'prop-types';
+
+const UploadContainer = styled.div`
+  display: inline-block;
+`;
+
+const Input = styled.input`
+  display: none;
+`;
+
+const Upload = ({
+  children,
+  clickArea = false,
+  droppable,
+  name,
+  accept,
+  value,
+  onChange,
+  ...props
+}) => {
+  const [file, setFile] = useState(value);
+  const [dragging, setDragging] = useState(false);
+  const inputRef = useRef(null);
+  const enterCounter = useRef(0);
+
+  const handleFileChange = (e) => {
+    const files = e.target.files;
+    const changedFile = files[0];
+    setFile(changedFile);
+    onChange && onChange(changedFile);
+  };
+
+  const handleChooseFile = () => {
```

Comment:
choose라는 유사 의미보다는 최대한 일관되게 이벤트명과 동일하게 가져가는것을 추천드립니다!
```suggestion
  const handleClickFile = () => {
```